source/likely.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
 var likely = require('./index.js');
+var history = require('./history');
+var utils = require('./utils');
 
 window.likely = likely;
-window.addEventListener('load', likely.initiate);
+window.addEventListener('load', function () {
+    if (likely.defaults.listenChangesUrl) {
```

Comment:
> Do you think it's redundant for a such plugin?

IDK. I’m trying to find the best balance between simple logic with no configuration and more complex logic with configuration. This is why I’m questioning the config usefulness: if the config won’t be used, there’s no need to keep it.

> Single Page Applications. Usually the one has complex logic and listens url changes by itself. It such case a user may want to run the process manually.

But do you know a concrete example when a user needs to disable history listening? I’m assuming that in most (if not all) cases, Likely will handle the URL changes absolutely OK. Is there a concrete case when Likely does it wrong, and the user needs to disable history listening? I can come up with theoretical cases, but they don’t seem real to me.

If you (just like I) don’t have any concrete examples, I’d remove this until there’s a real user request.